cmd/shim/main.go

```diff
@@ -277,7 +277,17 @@ func shim() int {
 	if err != nil {
 		panic(err)
 	}
-	cmd.Stdout = scrubOutWriter
+
+	outNlbw := NewLineBreakWriter(
```

Comment:
nit: this was a pre-existing issue, but I think it would make sense to have a fast path where if there are no secrets set in the exec, `cmd.Stdout`/`Stderr` get set directly to `outWriter`/`errWriter`. Just to avoid some of the potential overhead of the scrubbing code when it's not needed.